packages/seacas/libraries/ioss/src/Ioss_Utils.C

```diff
@@ -2009,6 +2052,605 @@ void Ioss::Utils::copy_database(Ioss::Region &region, Ioss::Region &output_regio
   }
 }
 
+bool Ioss::Utils::compare_database(Ioss::Region &input_region, Ioss::Region &output_region,
```

Comment:
Maybe put all of the compare functions in a `Ioss_Compare.{h|C}` instead of Ioss_Utils?  I know Ioss_Utils is already cluttered, but this adds a lot of code and might be the start of cleaning out Ioss_Utils.